resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/domain/ResourceEntity.java

```diff
@@ -39,11 +44,15 @@
                 @Index(name = "resource_id", columnList = "resource_id", unique = true),
                 @Index(name = "parent_id", columnList = "parent_id"),
         })
-public class ResourceEntity {
+@RedisHash("ResourceEntity")
+public class ResourceEntity implements Serializable {
     @Id
+    @org.springframework.data.annotation.Id
```

Comment:
What's the need to have both of these Id annotations?

Reply:
Redis requires a separate set of spring annotations to h2 and javax annotations